docs/.vuepress/sidebar/en.ts

```diff
@@ -120,7 +119,12 @@ export const enSidebar = sidebar({
     {
       text: "Java SDK", // required
       link: "/api/java/getting-started", // optional, which should be a absolute path.
```

Comment:
```suggestion
      link: "/api/java", // optional, which should be a absolute path.
```
To point to the README file